File: lib/go/invinput/missing_argument.go
// https://github.com/Zeronetsec/Ares

package invinput

import (
    "fmt"
    "github.com/Zeronetsec/Ares/lib/go/color"
)

func MissingArgument(command string) {
    fmt.Printf(
        "%s[!] %sMissing argument!\n",
        color.R, color.N,
    )

    fmt.Printf(
        "%s[!] %sTry: %saresdoc %s%s%s\n",
        color.R, color.N, color.GG, color.CC, command, color.N,
    )
}

// Copyright (c) 2026 Zeronetsec

File: lib/python/missing_argument.py
# https://github.com/Zeronetsec/Ares

import sys
import os

pylib = os.getenv("__lib__")
sys.path.append(f"{pylib}/python")

import color

def execute(command):
    print(f"{color.R}[!] {color.N}Missing argument!")
    print(f"{color.R}[!] {color.N}Try: {color.GG}aresdoc {color.CC}{command}{color.N}")

# Copyright (c) 2026 Zeronetsec